lib/driver.js

```diff
@@ -332,20 +333,14 @@ class XCUITestDriver extends BaseDriver {
       await this.stopRemote();
     }
 
-    await this.runReset();
-
     if (this.isSimulator() && this.opts.udid && this.opts.customSSLCert) {
       await uninstallSSLCert(this.opts.customSSLCert, this.opts.udid);
     }
 
     if (this.isSimulator() && !this.opts.noReset && !!this.opts.device) {
-      log.debug('Resetting simulator');
-      if (this.lifecycleData.bootSim) {
```

Comment:
Why is this being removed? Currently if we start a simulator we end by closing it. If we're changing this behavior it needs to be noted. And the rest of the mechanics of keep track needs to be yanked out.